src/app/domain/sharing/services/SharingNetwork.ts

```diff
@@ -0,0 +1,48 @@
+import CozyClient, { generateWebLink, Q } from 'cozy-client'
+
+import { sharingLogger } from '..'
+import { SharingCozyApp } from '../models/SharingCozyApp'
+
+export const fetchSharingCozyApps = {
+  definition: Q('io.cozy.apps').where({
+    'accept_documents_from_flagship.route_to_upload': { $exists: true },
```

Comment:
You should not add the `$exists:true` but instead you should index on this attribute. 

https://docs.cozy.io/en/tutorials/data/advanced/#indexes-why-is-my-document-not-being-retrieved